cli/cmd/quickStart.go

```
@@ -0,0 +1,161 @@
+			fmt.Printf("📦 Pulling rawpair/%s:latest...\n", name)
+			cmd := exec.Command("docker", "pull", fmt.Sprintf("rawpair/%s:latest", name))
+			cmd.Stdout = os.Stdout
+			cmd.Stderr = os.Stderr
+			if err := cmd.Run(); err != nil {
+				fmt.Printf("Failed to pull rawpair/%s: %v\n", name, err)
+			}
+		}
+
+		for _, label := range selectedTagOptions {
+			if tag, ok := labelMap[label]; ok {
+				selectedTags = append(selectedTags, tag)
```

golangci-lint (1.64.8)

SA4010: this result of append is never used, except maybe in other appends

(staticcheck)